aten/src/ATen/core/Tensor.h

```diff
@@ -183,7 +183,7 @@ class CAFFE2_API Tensor {
   }
 
   at::MemoryFormat suggest_memory_format() const {
-    if (impl_->is_strides_like_channels_last()) {
+    if (get_memory_format_propagation() && impl_->is_strides_like_channels_last()) {
```

Comment:
So... this is reverting a change that you made previously?

Reply:
Will likely be solved through a merge conflict later, but just to point out the potential issue that was addressed in my pending PR: https://github.com/pytorch/pytorch/pull/23861/files#diff-4632522f237f1e4e728cb824300403acR188